force-app/main/default/classes/LeadTriggerHandler.cls

```diff
@@ -32,17 +32,17 @@ public with sharing class LeadTriggerHandler {
 	 */
 	public static void handleTitleNormalization(List<Lead> leadsToNormalize) {
 		for (Lead ld : leadsToNormalize) {
-			if (ld.title == 'vp' || ld.title.contains('v.p.') || ld.title.contains('vice president')) {
+			if (ld.Title == null) {
+				continue;
+			} 
+			
+			if (ld.Title.contains('vp') || ld.Title.contains('v.p.') || ld.Title.contains('vice president')) {
```

Comment:
Try using a list and check if the values are in the list. This way you can add more values and not need to modify the logic.